### PR TITLE
Fixed a dependency constraint issue with the package:web

### LINKED ISSUE #454: Update intercom_flutter_web dependency to web 1.0 (now web 0.x)
It's incompatible with other libraries' last versions

### PR DESCRIPTION
**Pull request by vlazdra**
Fixes #454 
Fixes the issue with dependency constraint of the `package:web` since a newer version has been released.

I've ran the `intergration_tests` in the web package folder and all of them passed. It seems that no other changes are needed.

### CODE REVIEW

**Comment by vlazdra** [comment]
@deepak786 Can you please review and make a new release afterwards? Thank you! :)

**Comment by dario-valles** [comment]
Please review and merge!

**Comment by Zazo032** [comment]
Can you also bump the package’s `version` and add it to the changelog?

**Comment by vlazdra** [comment]
@Zazo032 Done.

**Review by deepak786 — APPROVED** [review]

File: intercom_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 9.0.10
+* Bump `intercom_flutter_web` to `1.1.2` supporting `web` `^1.0.0`.
```
Comment (deepak786) [inline intercom_flutter/CHANGELOG.md:5]: Add linebreak between

File: intercom_flutter_web/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.1.2
+* Bump `web` to `^1.0.0`
```
Comment (deepak786) [inline intercom_flutter_web/CHANGELOG.md:5]: Add linebreak between

**Review by deepak786 — COMMENTED** [review]
I have published the `intercom_flutter_web: 1.1.2` but check my comments so that I can merge this. After that I will update intercom_flutter.

**Comment by vlazdra** [comment]
@deepak786 Done.

**Pull request merged**